.travis.yml

```diff
@@ -1,9 +1,6 @@
 language: node_js
 node_js:
-- '9'
+- '10'
 cache:
   directories:
     - node_modules
-before_script: 
-- cd node_modules/truffle && npm install solc@0.5.0 && cd ../..
-- truffle compile
```

Comment:
So no more truffle compile?

Also nit, no new line at EOF

Reply:
truffle compile is part of truffle test.